styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -120,12 +121,14 @@
   public static final String STYX_MODE_DEVELOPMENT = "development";
 
   public static final int SCHEDULER_TICK_INTERVAL_SECONDS = 2;
+  public static final int SUBMISSION_RATE_LIMITER_UPDATE_TICK_INTERVAL_SECONDS = 5;
```

Comment:
@ulzha Do you mean that you'd rather have `startSubmissionRateLimiterUpdate()` be named something like `startConfigUpdate()` ?